### PR TITLE
Update boost proposal with details

### PR DESCRIPTION
**Pull request by AGaliuzov**
Update proposal with the following information
1. Add exact version of the library
2. Update impact on the existing code. Add code snippets and describe the drawbacks and advantages.

### CODE REVIEW

**Comment by AGaliuzov** [comment]
@theresalech, @joeljfischer please let me know of this pull request contains enough information 
to proceed further with the proposal.

File: proposals/0044-use-Boost-library.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ From time to time we’re facing with different problems with our [Utils](https:
 * All the time we have to solve customer’s problems very fast and we have no enough time for current utilities adaptation. Usually it much faster to develop necessary functionality from the scratch. But such approach brings us new defects, waste our time to support and improve this functionality. 
 
 ## Proposed solution
-The porposed solution which has to adress mentioned issues is to use [Boost library](http://www.boost.org/users/history/version_1_62_0.html)
+The porposed solution which has to adress mentioned issues is to use [Boost library v1.56](http://www.boost.org/users/history/version_1_56_0.html)
```
Comment (theresalech) [inline proposals/0044-use-Boost-library.md]: Please update "porposed" to "proposed" and "adress" to "address"

File: proposals/0044-use-Boost-library.md [hunk 2]
```diff
@@ -41,11 +41,125 @@ The porposed solution which has to adress mentioned issues is to use [Boost libr
 This is additional dependency which is out of our control. It's not easy to apply possible customization if required - it has to be accepted by Boost community. In the worst case we might fork the whole library and customize it on our own.
 
 ## Impact on existing code
-This proposal is not geared to provide any changes to the existed code. It rather allows to share the common idea is how greate to have such library as Boost. Once it will be accepted there some number of separate proposals will be created. One proposal for one improvement using the Boost library.
-* Change thread managment in SDL.
-* Use the filesystem api from the Boost instead of one developed within SDL.
-* Substitute SDL's timers with Boost analog.
-* Substitute SDL's `DateTime` with Boost analog 
+Every utility component such as `Thread`, `Filesystem`, `DateTime`, `Timer` should be changed. As the good example of the possible changes propose to start from the `Filesystem` component. Below you can find the current imiplementation
```
Comment (theresalech) [inline proposals/0044-use-Boost-library.md]: Please change "imiplementation" to "implementation"

**Comment by joeljfischer** [comment]
Is this a replacement for #148?

**Comment by AGaliuzov** [comment]
@theresalech done.

**Comment by AGaliuzov** [comment]
@joeljfischer yes, this is the replacement. I will ask the PR owner to close the #148.

**Pull request merged**